Sources/Wrap.swift

```diff
@@ -120,13 +120,13 @@ public protocol WrapCustomizable {
      *  Returning nil from this method will be treated as an error, and cause
      *  a `WrapError.WrappingFailedForObject()` error to be thrown.
      */
-    func wrap() -> AnyObject?
+    func wrap(context: Any?) -> AnyObject?
```

Comment:
I'm thinking if we want to use explicit parameter labeling here (Swift 3-style) to make the API more intuitive. Right now I have to know that the object I supply refers to `context`, but if we use a parameter label it'll be a lot more clear, like this:
```
// When used without the context parameter
wrap()
```
```
// When used with the context parameter
wrap(context: "A context")
```
What do you think @Schaltfehler?